certified-connectors/Elfsquad Data/apiDefinition.swagger.json

```diff
@@ -194,6 +194,27 @@
         "x-ms-visibility": "internal"
       }
     },
+    "/data/1/{entity_name}/$patch-schema": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@thomasio101 default responses are not allowed. please define a proper [response](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
@Amjed-Ayoub, duly noted!